src/main/java/org/jenkinsci/plugins/github_branch_source/BranchDiscoveryTrait.java

```diff
@@ -52,6 +52,23 @@
  * @since 2.2.0
  */
 public class BranchDiscoveryTrait extends SCMSourceTrait {
+    /**
+     * None strategy.
+    */
+    public static final int NONE = 0;
+    /**
+     * Exclude branches that are also filed as PRs.
+    */
+    public static final int EXCLUDE_PRS = 1;
+    /**
+     * Only branches that are also filed as PRs.
+    */
+    public static final int ONLYP_RS = 2;
```

Comment:
shouldn't this be `ONLY_PRS`?

Reply:
🤦‍♂️ https://github.com/jenkinsci/github-branch-source-plugin/pull/377

Reply:
My bad, I should have caught that.